### ISSUE TITLE
Failing to run with io.js or Node.js 0.12

### ISSUE THREAD

**Issue by shama** [node-inspector/node-inspector#657]
After trying `npm i -g node-inspector` and `node-debug index.js` where `index.js` only contains `console.log('test')` I get a blank source and the script appear to fail to run:

![node-inspector](https://cloud.githubusercontent.com/assets/99604/7850396/b0d33344-0496-11e5-99b6-27e92d3e5c19.gif)

I'm on OSX 10.10.3 using Chrome 43.0.2357.81
I tried with io.js `2.1.0` and `1.8.2` and Node.js `0.12.4` (and re-installing node-inspector each switch).
I tried node-inspector `0.10.1` and `0.8.3`

I also tried with `--target` doing: `npm i -g node-inspector --target=1.8.2` (which installed with `--target=1.8.2` but still had the same issue; it did not even install using `--target=2.1.0`).

---

The only way I was able to get it working was by installing Node.js `0.10.38` and node-inspector `0.10.1`.

Thanks for your help!

**Comment by 3y3** [comment]
Hello @shama , you can do the following:
1. Run `node-inspector` in first shell and `node --debug-brk test` in second.
Is there any errors in first shell?
2. If there is no error in first step, can you please follow [this instruction](https://github.com/node-inspector/node-inspector/wiki/Developer%27s-Guide#reverse-engineering-the-front-end)
3. How long app starts up on 0.10.38?

**Comment by shama** [comment]
@3y3 Thanks for your help!

1. Ran both of those and no errors in the first shell (running `node-inspector`)
2. Here is the output I get running `DEBUG=node-inspector:protocol:* node-inspector`: https://gist.github.com/shama/ebc5063878499422b273
3. It's not immediate but the source appears ~ 2 seconds.

**Comment by 3y3** [comment]
Hm... This looks like a problem with connection to debugged app.
I receive the same trace log, if I run node-inspector without debugged app.
I don't know, what else I need at current time for debugging...
Are you ready to modify node-inspector sources?

**Comment by shama** [comment]
@3y3 Sure! I'll dive into the source and see if I can find out where it's failing.

**Comment by jmm** [comment]
I don't know for sure if I'm having the same problem, but it seems similar. node-inspector used to work great but now it's doing this thing most of the time where the interface loads but does nothing, remains completely blank. My environment:

node 0.10.32 / node-inspector 10.0.0 / 10.0.1 running on Ubuntu 12.04.
Debugging remotely from Windows 7 x64 / Chrome 42 on different physical machine on same local network.
```sh
node-debug --debug-brk --web-host=example.com script.js
```

Since I updated Chrome to v41 it started exhibiting frequent DNS failures and I recently ran Fiddler (debugging proxy) to see if it could reveal anything, and just running it seems to sidestep the Chrome DNS failures, so I've been running it since. I mention it in case it's relevant somehow, but as I said the interface loads, just nothing within it. And I believe I've had it function normally at least once with Fiddler running.

**Comment by jmm** [comment]
I don't know if it's relevant, but I'm also getting this in Chrome console:
```
Assertion failed: Unknown experiment canvasInspection
  Runtime._assert @ Runtime.js:358
  Runtime.ExperimentsSupport._checkExperiment @ Runtime.js:953
  Runtime.ExperimentsSupport.isEnabled @ Runtime.js:894
  WebInspector.ProfileTypeRegistry @ ProfileTypeRegistry.js:19
  (anonymous function) @ ProfileTypeRegistry.js:43
  evaluateScript @ Runtime.js:153
  scriptSourceLoaded @ Runtime.js:136
```

**Comment by 3y3** [comment]
@jmm , do you have `fiddler` enabled permanently?
I know one shadow issue about `fiddler + web sockets`.

**Comment by jmm** [comment]
@3y3 Thanks for the reply. Lately I've been running `fiddler` continuously. I'm not sure if that's what you mean by permanently. Chrome is so messed up that if I disable `fiddler` and try to open the `node-inspector` URL, I immediately get this `net::ERR_NAME_NOT_RESOLVED` error that I get rampantly (since moving from Chrome v36 [I think] to v41) when `fiddler` isn't running.

> I know one shadow issue about `fiddler + web sockets`.

Do you know of something I can / should do to configure `fiddler` to work with `node-inspector`? Would following those instructions you linked to shed more light on my problem? I believe I've had it work at least once with `fiddler` running, but I can't say for certain.

**Comment by 3y3** [comment]
> I immediately get this net::ERR_NAME_NOT_RESOLVED error that I get rampantly (since moving from Chrome v36 [I think] to v41) 

Can you give me a screenshot with this error.

> Do you know of something I can / should do to configure fiddler to work with node-inspector? 

Unfortunately I don't know any solution here.

I also wonder to see [debug log](https://github.com/node-inspector/node-inspector/wiki/Debug-logs) . It will be helpful.

**Comment by jmm** [comment]
@3y3 Thanks very much for your help. It sounds like the same symptoms, but in case the cause of my problem is different I don't want to squat on @shama's issue, so I opened #671 to pursue mine further. I started with a recap of the basic info I posted here and I'll be posting additional information.

**Comment by shama** [comment]
Closing as this seems to be fixed with node-inspector `0.12.3` and node.js `4.0.0`. Thanks!
